src/07_pakages/task_01_createpakage/demo.py

```diff
@@ -0,0 +1,48 @@
+"""Demonstrate the pakages by doing different operations."""
```

Comment:
```suggestion
"""Demonstrate the packages by doing different operations."""
```

Avoid spell mistakes